nuxeo-compound-documents/src/main/resources/OSGI-INF/service-contrib.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.compound.documents.service">
+  <documentation>
+    Compound Documents Manager, delegating logic to the appropriate compound document type.
```

Comment:
Is this accurate on what the service does? I'm not sure :)

Reply:
won't be me if I didn't let one slip...